juju/juju
# InstanceMutator Machine API for SetModificationStatus

## Description of change

The following exposes SetModificationStatus on the instance mutator
machine api.

## QA steps

Note to actually see the changes, edit the `worker/instancemutater/mutater.go`
file and add a call to the `machine.SetModificationStatus`

Then perform the following:

```
juju bootstrap lxd test
juju deploy ./testcharms/charm-repo/quantal/lxd-profile
```
